columnize.go

```diff
@@ -65,18 +67,22 @@ func getWidthsFromLines(config *Config, lines []string) []int {
 // returns a sprintf-style format string which can be used to print output
 // aligned properly with other lines using the same widths set.
 func (c *Config) getStringFormat(widths []int, columns int) string {
+	// Create the buffer
+	// 8 bytes per character * 8 characters per column * ~num columns
+	buf := bytes.NewBuffer(make([]byte, 0, 8*len(widths)*8))
+
 	// Start with the prefix, if any was given.
-	stringfmt := c.Prefix
+	buf.WriteString(c.Prefix)
```

Comment:
Since this is well documented and we aren't actually skipping any important errors, lets just put this back the way you had it. I think we are good for a merge after that.